WireAnalytics/Sources/WireAnalytics/Events/Segmentation/Predefined/Segmentation+Unspecific.swift

```diff
@@ -55,16 +55,13 @@ extension SegmentationEntry {
     /// - Returns: A `SegmentationEntry` instance with the appropriate key and value.
 
     static func isVideoCall(_ value: Bool) -> Self {
-        .init(key: "is_video_call", value: value)
+        .init(key: "call_video", value: value)
```

Comment:
Is there a reason why this isn't scoped under the `Call` namespace?

Reply:
Good catch, this is a mistake. I'll move it there.